test/src/engine/BMGameTest.php

```diff
@@ -8827,12 +8827,10 @@ public function test_mood_swing_rounds() {
         $this->assertInstanceOf('BMDieSwing', $button1->dieArray[4]);
         $this->assertTrue($button1->dieArray[4]->needsSwingValue);
         $this->assertTrue($button1->dieArray[4]->has_skill('Mood'));
-        $this->assertEquals(array('pre_roll', 'add_skill'),
+        $this->assertEquals(array('pre_roll'),
```

Comment:
I also removed the die hook add_skill (and associated functions) from Mood because it wasn't doing anything.